selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestAccountFields.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.pageobject.dashboard.account;
+
+public class TestAccountFields {
```

Comment:
It's just regular account fields. Nothing special. So, let's name this class as just "Account".